clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -188,6 +187,11 @@ public String toString() {
         private LinkedHashMap<TopicPartition, PartitionData> next;
         private final boolean copySessionPartitions;
 
+        private List<TopicPartition> added = new ArrayList<>();
+        private List<TopicPartition> removed = new ArrayList<>();
+        private List<TopicPartition> altered = new ArrayList<>();
+
+
         Builder() {
             this.next = new LinkedHashMap<>();
```

Comment:
Should we rename next now? I think the name previously made sense because sessionPartitions would be pointed at it after each pass, so it was the "next" sessionPartitions, but now it's being used as an "updated" hashmap, and is completely thrown away after each pass.